zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/exceptions/ExporterElasticsearchException.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.exceptions;
+
+public class ExporterElasticsearchException extends RuntimeException {
+  public ExporterElasticsearchException(final String message) {
```

Comment:
I'd override the constructor that receives both the message and the throwable (not to swallow the root cause here)
```java
public ElasticsearchExporterException(final String message, final Throwable cause) {
    super(message, cause);
  }
```